### PR TITLE
Add release notes for Frontend v3.10.0

### PR DESCRIPTION
**Pull request by EoinShaughnessy**
Fixes https://github.com/alphagov/govuk-frontend/issues/1986

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,13 +2,21 @@
...
+#### Update users with notification banners
+
+You can now [use notification banners](https://design-system.service.gov.uk/components/notification-banner/) to inform users about events that occur elsewhere in the service. For example, when they need to complete tasks.
+
+This was added in [pull request #1935: Create notification banner component](https://github.com/alphagov/govuk-frontend/issues/1935).
+
 #### Customise input mode in the date component
 
-You can now customise the input mode of each input in the date component to trigger a keyboard of your choice to appear on mobiles.
+You can now enable mobile users to change their keyboard selection when they complete fields in the [date component](https://design-system.service.gov.uk/components/date-input/).
```
Comment (EoinShaughnessy) [inline CHANGELOG.md:17]:
```suggestion
You can now change which keyboard type displays for mobile and tablet users. This enables users to enter characters other than numbers when they use the date input. To change the keyboard type, set the `inputmode` when you use the Nunjucks macro options.```

**Review by 36degrees — APPROVED** [review]
Looks good to me – thanks! 👍🏻

**Comment by EoinShaughnessy** [comment]
Have sent release notes for tech writer review.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,13 +2,21 @@
 
 ## Unreleased
 
+## 3.10.0 (Feature release)
+
 ### New features
 
+#### Update users with notification banners
+
+You can now [use notification banners](https://design-system.service.gov.uk/components/notification-banner/) to inform users about events that occur elsewhere in the service. For example, when they need to complete tasks.
```
Comment (EoinShaughnessy) [inline CHANGELOG.md:11]:
```suggestion
You can now [use notification banners](https://design-system.service.gov.uk/components/notification-banner/) to tell users about events that occur elsewhere in your service. For example, when they need to complete tasks.
```

**Comment by EoinShaughnessy** [comment]
Steve Evans has given 2i approval.

**Pull request merged**